src/auth/auth.service.ts

```diff
@@ -3,29 +3,38 @@ import { TokenPayload } from './interfaces/token-payload.interface';
 import { JwtService } from '@nestjs/jwt';
 import { v4 as uuidv4 } from 'uuid';
 import { AuthDTO } from './dto/auth.dto';
+import { AstraService } from '../astra/astra.service';
 
 @Injectable()
 export class AuthService {
   //TODO move configCollection to database => own ConfigDataModule
   private configCollection: { [id: string]: { knownClients: string[] } } = {};
-  constructor(private readonly jwtService: JwtService) {}
+  constructor(
+    private readonly jwtService: JwtService,
+    private readonly astraService: AstraService,
+  ) {}
 
-  public register(body: AuthDTO) {
+  public async register(body: AuthDTO) {
     const clientId = uuidv4();
     const { serverId, scopes } = body;
+    let tokens: String[] = [];
+    try {
+      tokens = await this.astraService
+        .get<String[]>('Benjamin', serverId, 'tokens')
```

Comment:
@Cahllagerfeld was `Benjamin` part of testing?

Reply:
Oops I see you did another commit afterwards, may bad 🤦‍♂️ - just trying to work through conflicts